src/server.h

```diff
@@ -141,6 +143,21 @@ struct ClientInfo {
 	std::string vers_string, lang_code;
 };
 
+struct ModIPCStore {
+	/// RW lock for this entire structure
+	std::shared_mutex mutex;
+	/// Signalled on any changes to the map contents
```

Comment:
Why does this line and 147 use three `///`

Reply:
doxygen